node/utils/message.ts

```diff
@@ -86,6 +88,13 @@ const sendMailNotificationToUsers = async (
           templateName,
         })
       )
+
+      sendMessageMetric({
+        quote,
+        account: sender.context.account,
+        sentTo: user,
+        templateName,
+      })
```

Comment:
For analytics/query purposes, I believe saving one line per user is better because it's faster/easier to query distinct values than using string search (with `like` and %). I meant, if we need to count messages by quote or by receiver, it's faster if we duplicate lines.
I'll do changes as you recommended.